framework/src/meshgenerators/PatternedMeshGenerator.C

```diff
@@ -150,6 +150,19 @@ PatternedMeshGenerator::generate()
       // Move the mesh into the right spot.  -i because we are starting at the top
       MeshTools::Modification::translate(cell_mesh, deltax, -deltay, 0);
 
+      // Define a reference map variable for subdomain map
+      auto & main_subdomain_map = _row_meshes[i]->set_subdomain_name_map();
+      // Retrieve subdomain name map from the mesh to be stitched and insert it to the main
+      // subdomain map
+      const auto & increment_subdomain_map = cell_mesh.get_subdomain_name_map();
+      main_subdomain_map.insert(increment_subdomain_map.begin(), increment_subdomain_map.end());
+      // Check if one SubdomainName is shared by more than one subdomain ids
+      std::set<SubdomainName> main_subdomain_map_name_list;
```

Comment:
it's good to comment every line but what's the goal here?
The check on subdomain ids?